actions/check-for-changeset/action.yml

```diff
@@ -5,6 +5,10 @@ inputs:
     description: list of comma-separated paths (files or directories ending in /) that don't need a changeset when changed. Default .github/
     required: false
     default: .github/
+  exclude_extensions:
+    description: a list of comma-separated extensions that don't need a changeset when changed. Default <empty>
+    required: false
+    default: ''
```

Comment:
oh you forgot `exclude_glob` here too I think?

Reply:
Oops... I left `exclude_glob` in by accident.  I saw we don't currently have any sort of `glob` dependency and was wondering if that'd be ok to add.  If so, I'll add it and plumb support through.